REQUIRE

```diff
@@ -1,4 +1,4 @@
 julia 0.4
 URIParser
 SHA
-Compat 0.8.4
+Compat 0.9.5
```

Comment:
by making the returned code version-dependent we've removed the need to bump the compat minimum